netbeans-plugin/pom.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Missing license header (please insert  the correct License header!!). Also plugin versions must be specified into parent pom.xml and then used into child pom.xml